code/modules/reqs/supplypacks.dm

```diff
@@ -34,13 +34,21 @@ OPERATIONS
 /datum/supply_packs/operations/beacons_supply
 	name = "supply beacon"
 	contains = list(/obj/item/squad_beacon)
-	cost = 20
+	cost = 10
 
 /datum/supply_packs/operations/beacons_orbital
 	name = "orbital beacon"
 	contains = list(/obj/item/squad_beacon/bomb)
 	cost = 30
 
+/datum/supply_packs/operations/fulton_extraction_pack
+	name = "fulton extraction pack"
+	contains = list(
+		/obj/item/fulton_extraction_pack,
+		/obj/structure/fulton_extraction_point
+	)
```

Comment:
```suggestion
	name = "fulton extraction pack"
	contains = list(/obj/item/fulton_extraction_pack)
```

Reply:
The fulton_extraction_point is a beacon to which you link the fulton, like how medevac beds work. From what I'm reading, you can fulton off xenos corpses without it for points, but this just gives another way to move stuff around. If people start using it en-masse as a cheaper medibed (if that's even possible), I'll punt it.

Reply:
It's a deprecated item, Fultons autosell their attached targets

Reply:
based